### PR TITLE
fix(TCOMP-2564): component-starter image startup hangs

### PR DESCRIPTION
**Pull request by undx**
https://jira.talendforge.org/browse/TCOMP-2564

### CODE REVIEW

**Review by ozhelezniak-talend — APPROVED** [review]
lgtm

File: component-starter-server/src/main/java/org/talend/sdk/component/starter/server/service/info/ServerInfo.java [hunk 1]
```diff
@@ -159,7 +159,7 @@ private void refresh() {
             parser.parse(metadata, handler);
             if (handler.release != null) {
                 final String release = handler.release.toString();
-                if (!release.equals(snapshot.getKit())) {
+                if (!release.equals(snapshot.getKit()) && !"10.57.0".equals(release)) {
```
Comment (ozhelezniak-talend) [inline component-starter-server/src/main/java/org/talend/sdk/component/starter/server/service/info/ServerInfo.java:162]: What's the magic constant?

File: images/component-starter-server-image/pom.xml [hunk 2]
```diff
@@ -89,9 +89,7 @@
                   <MEECROWAVE_BASE>${docker.talend.dir.app}</MEECROWAVE_BASE>
                   <MEECROWAVE_PID>${docker.talend.dir.app}conf/server.pid</MEECROWAVE_PID>
                   <MEECROWAVE-PROPERTIES>${docker.talend.dir.app}conf/meecrowave.properties</MEECROWAVE-PROPERTIES>
-                  <TALEND_JDK_SERIAL_FILTER>oracle.sql.**;!*</TALEND_JDK_SERIAL_FILTER>
```
Comment (ozhelezniak-talend) [inline images/component-starter-server-image/pom.xml:92]: I remember that we needed it for some bug...

File: component-starter-server/src/main/java/org/talend/sdk/component/starter/server/service/info/ServerInfo.java (hunk 1 above)
Comment (undx) [reply component-starter-server/src/main/java/org/talend/sdk/component/starter/server/service/info/ServerInfo.java:162]: This is not a magic constant, we want to exclude this release. It was released by Axel for testing stuffs and we cannot delete it and unfortunately it is considered as latest in maven: see https://repo.maven.apache.org/maven2/org/talend/sdk/component/component-starter-server/maven-metadata.xml

File: images/component-starter-server-image/pom.xml (hunk 2 above)
Comment (undx) [reply images/component-starter-server-image/pom.xml:92]: It was for component-server, not component-starter-server.

**Pull request merged**